server/controllers/packFeed.js

```diff
@@ -0,0 +1,90 @@
+const db = require('../db/database');
+var getUserPacksId = (userId, res) => {
+  // db.connect();
+  //replace below query with: `select * from public."packUsers" where public."packUsers"."userId" = ${userId} or public."packUsers"."userId" = 15`
+  //to test if it works with multiple groups being returned by query.
```

Comment:
no worries about changing this, but just flagging for future ref that the variable names here are a bit unclear - would be better if more descriptive of the data at each step

Reply:
Will look into it, Ill submit the next pull request without this change for the sake of time but the next one after that will have updated variable names.